src/mono/wasm/debugger/BrowserDebugProxy/ValueTypeClass.cs

```diff
@@ -110,17 +111,39 @@ JObject GetFieldWithMetadata(FieldTypeClass field, JObject fieldValue, bool isSt
                 return fieldValue;
             }
         }
+        public async Task<string> ToString(MonoSDBHelper sdbAgent, CancellationToken token)
+        {
+            var typeInfo = await sdbAgent.GetTypeInfo(TypeId, token);
+            if (typeInfo == null)
+                return null;
+            if (typeInfo.Name == "object")
+                return null;
+            Microsoft.WebAssembly.Diagnostics.MethodInfo methodInfo = typeInfo.Info.Methods.FirstOrDefault(m => m.Name == "ToString");
+            if (!IsEnum && methodInfo == null)
+                return null;
+            int[] methodIds = await sdbAgent.GetMethodIdsByName(TypeId, "ToString", token);
+            if (methodIds == null)
+                return null;
+            try {
+                var retMethod = await sdbAgent.InvokeMethod(Buffer, methodIds[0], token, "methodRet");
```

Comment:
There could be other `ToString` methods like too `public string ToString(int x) => $"{x+2}";`, so we need to find the one that has no parameters.

Reply:
Done